.github/workflows/validations.yml

```
@@ -11,93 +11,186 @@ jobs:
-          DB_PASS: "password"
-          TEST_ENV_NUMBER: ${{ github.event.number }}
+          DB_PASS: password
+          CI_NODE_INDEX: ${{ matrix.ci_node_index }}
+          CI_NODE_TOTAL: ${{ matrix.ci_node_total }}
         run: |
-          bundle exec rails db:prepare
-      - name: Run Rubocop
+          bundle exec rails db:create
+          bundle exec rails db:schema:load
+          bundle exec rails db:migrate
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)